src/main/java/com/google/jenkins/plugins/computeengine/CleanLostNodesWork.java

```diff
@@ -39,11 +40,15 @@
 @Symbol("cleanLostNodesWork")
 public class CleanLostNodesWork extends PeriodicWork {
     protected final Logger logger = Logger.getLogger(getClass().getName());
+    public static final String NODE_IN_USE_LABEL_KEY = "jenkins_node_in_use";
+    public static final long RECURRENCE_PERIOD = Long.parseLong(
+            System.getProperty("jenkins.cloud.gcp.cleanLostNodesWork.recurrencePeriod", String.valueOf(HOUR)));
+    public static final int LOST_MULTIPLIER = 3;
```

Comment:
Why `public`?

Reply:
this constant is also used in the integration test class, but current convention for writing tests is in a subpackage `integration`,
so `com.google.jenkins.plugins.computeengine.integration.CleanLostNodesWorkIT`

then can't access with even package private and `@VisibleForTesting`

Reply:
OK, maybe add `@VisibleForTesting` here for clarity.